.github/workflows/build-desktop-apps.yml

```diff
@@ -20,11 +20,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, macos-latest]
+        os: [ubuntu-latest, macos-14]
         include:
           - os: ubuntu-latest
             platform: linux
-          - os: macos-latest
+          - os: macos-14
```

Comment:
what's wrong with latest?

Reply:
I want to build it on runners running Apple silicon (much, much faster), and its on `macos-14` only for now. See [this](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories)

Reply:
I didn't know that! Thanks for sharing.